### PR TITLE
Actions playwright setup

### PR DESCRIPTION
**Pull request by JordanWestern**

### CODE REVIEW

File: Plugins/SpecFlow.Actions.Playwright/Example/PageObjects/CalculatorPageObject.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        private async Task<IPage> CreatePageAsync(Task<IBrowser> browser)
+        {
+            return await (await browser).NewPageAsync();
+        }
+
+        public async Task GoToPage()
+        {
+            await (await _page).GotoAsync(CalculatorUrl);
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/Example/PageObjects/CalculatorPageObject.cs:35]: Let's wrap this into a function to make it more readable.

File: Plugins/SpecFlow.Actions.Playwright/Example/Steps/CalculatorStepDefinitions.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        //Page Object for Calculator
+        private readonly CalculatorPageObject _calculatorPageObject;
+
+        public CalculatorStepDefinitions(CalculatorPageObject calculatorPageObject)
+        {
+            _calculatorPageObject = calculatorPageObject;
+        }
+
+        [Given("the first number is (.*)")]
+        public async Task GivenTheFirstNumberIsAsync(int number)
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/Example/Steps/CalculatorStepDefinitions.cs:20]: we could make this parameters to string and save the ToString() call

File: Plugins/SpecFlow.Actions.Playwright/SpecFlow.Actions.Playwright/NoopWebdriver.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Playwright;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SpecFlow.Actions.Playwright
+{
+    public class NoopWebdriver /*: IBrowser*/
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/SpecFlow.Actions.Playwright/NoopWebdriver.cs:8]: Dead code?

File: Plugins/SpecFlow.Actions.Playwright/SpecFlow.Actions.Playwright/Browser.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿namespace SpecFlow.Actions.Playwright
+{
+    public enum Browser
+    {
+        None = 0,
+        Chromium = 1,
+        Firefox = 2,
+        Edge = 3,
+        Chrome = 5,
+        Noop = 6
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/SpecFlow.Actions.Playwright/Browser.cs:10]: the noop value is now not necessary more, or?

File: Plugins/SpecFlow.Actions.Playwright/Example/Example.csproj [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    <PackageReference Include="SpecFlow.Plus.LivingDocPlugin" Version="3.*" />
+    <PackageReference Include="SpecFlow.xUnit" Version="3.*" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/Example/Example.csproj:18]: white space ;-)

File: Plugins/SpecFlow.Actions.Playwright/Example/Example.csproj [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    <ProjectReference Include="..\SpecFlow.Actions.Playwright\SpecFlow.Actions.Playwright.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="specflow.actions.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Playwright/Example/Example.csproj:37]: white space ;-)

File: Plugins/SpecFlow.Actions.Playwright/Example/PageObjects/CalculatorPageObject.cs (hunk 1 above)
Comment (JordanWestern) [reply Plugins/SpecFlow.Actions.Playwright/Example/PageObjects/CalculatorPageObject.cs:35]: what do you suggest?

Comment (SabotageAndi) [reply Plugins/SpecFlow.Actions.Playwright/Example/PageObjects/CalculatorPageObject.cs:38]: stupid async/await. lets keep it this way

**Review by SabotageAndi — APPROVED** [review]

**Pull request merged**